DDDEastAnglia/Areas/Admin/Models/SponsorModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Web;
+
+namespace DDDEastAnglia.Areas.Admin.Models
+{
+    public class SponsorModel
```

Comment:
This is just called `Sponsor` in the main site. Should we name this the same for consistency across the codebase and with the existing models?